packages/ui/src/components/templates/report/components/WebsiteCredibility/WebsiteCredibility.tsx

```
@@ -110,7 +110,7 @@ export const WebsiteCredibility: FunctionComponent<{
                 </ul>
               </ul>
             ))}
-          {isEmptyTrafficAnalysis && <>No Indications Detected.</>}
+          {isEmptyTrafficAnalysis && <>No traffic data detected.</>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)